server/config/routes.js

```diff
@@ -57,6 +54,7 @@ module.exports = function(app) {
     //needs to include all users for the story as well as tags/comments for that story
   app.get('/api/stories/:storyId', controller.stories.getOne);
 
+  app.post('/api/:storyId', controller.stories.filterByTag);
```

Comment:
What does this do? how does it work? what inputs is it expecting what output should i expect?